testutil/keeper/migration.go

```diff
@@ -33,19 +51,62 @@ func MigrationKeeper(t testing.TB) (keeper.Keeper, sdk.Context) {
 	cdc := codec.NewProtoCodec(registry)
 	authority := authtypes.NewModuleAddress(govtypes.ModuleName)
 
+	ctrl := gomock.NewController(t)
+	mockAccountKeeper := mocks.NewMockAccountKeeper(ctrl)
+
+	cfg := defaultConfigWithMocks(ctrl)
+	for _, opt := range opts {
+		opt(cfg)
+	}
+
 	k := keeper.NewKeeper(
 		cdc,
 		runtime.NewKVStoreService(storeKey),
 		log.NewNopLogger(),
 		authority.String(),
+		mockAccountKeeper,
+		cfg.bankKeeper,
 	)
 
 	ctx := sdk.NewContext(stateStore, cmtproto.Header{}, false, log.NewNopLogger())
 
 	// Initialize params
-	if err := k.SetParams(ctx, types.DefaultParams()); err != nil {
+	if err := k.SetParams(ctx, migrationtypes.DefaultParams()); err != nil {
 		panic(err)
 	}
 
 	return k, ctx
 }
+
+// WithBankKeeper assigns the given BankKeeper to the MigrationKeeperConfig.
+func WithBankKeeper(bankKeeper migrationtypes.BankKeeper) MigrationKeeperOptionFn {
+	return func(cfg *MigrationKeeperConfig) {
+		cfg.bankKeeper = bankKeeper
+	}
+}
+
+func defaultConfigWithMocks(ctrl *gomock.Controller) *MigrationKeeperConfig {
```

Comment:
#PUC what the default config provides.

In particular, the logic you have for SpendableCoins.